xde/src/xde.rs

```diff
@@ -1845,7 +1881,7 @@ fn next_hop(ip6_dst: &Ipv6Addr) -> (EtherAddr, EtherAddr) {
                 EtherAddr::zero(),
```

Comment:
Can also update this probe site (on the line above because github won't let me comment on it) with the source address (`src`) since we do have it at this point.

Reply:
Done